app/states/texas/validators/texas_filers.py

```
@@ -60,225 +79,275 @@ class TECFilerName(TECSettings, table=True):
-
-        _name = funcs.person_name_parser(f"{_first_name} {_last_name}")
-        values['filerNameLast'] = _name.last
-        values['filerNameFirst'] = _name.first
-        values['filerNameMiddle'] = _name.middle.replace('.', '')
-        values['filerNameSuffixCd'] = _name.suffix
-        values['filerNamePrefixCd'] = _name.title
-        values['filerNameFull'] = _name.full_name
-        return values
+    filer_name: Optional["TECFilerName"] = Relationship(back_populates="cta_details", link_model=TECFilerIDNameLink)
 
+    def __init__(self, **data):
```

Ruff (0.14.7)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)